src/orderly/run.py

```diff
@@ -40,26 +42,28 @@ def orderly_run(name, *, parameters=None, root=None, locate=True):
         raise Exception(msg) from error
 
     try:
-        _orderly_cleanup_success(packet, orderly)
+        _orderly_cleanup_success(packet, entrypoint, orderly)
     except Exception:
         _orderly_cleanup_failure(packet)
         raise
     return packet_id
 
 
-def _validate_src_directory(name, root):
+def _validate_src_directory(name, root) -> Tuple[Path, str]:
```

Comment:
we are really going to have to do a push through on adding these everywhere I think...